libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -32,7 +32,9 @@ export class StocksComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.stockPickerForm.valueChanges.subscribe(this.fetchQuote);
```

Comment:
This may not work. Changes might result in invoking fetchQuote API even if the user didn't  provided all the information on the form. Filter operator can be utilized to check validity of the form. 

filter( () =>  this.stockPickerForm.valid)

Once the form data is valid, stock price data can be fetched from the api.